src/utils/self_test/SConscript

```diff
@@ -13,15 +13,18 @@ SELF_TEST = 'self_test.c'
 def scons():
     """scons function"""
 
-    Import('env', 'prereqs')
+    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_utils_objs')
 
     tenv = env.Clone(LIBS=[])
 
+    tenv.AppendUnique(CPPPATH=['#/src/cart', '#/src/tests/ftest/cart'])
     tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm'])
     prereqs.require(tenv, 'mercury')
+    tenv.AppendUnique(LIBPATH=['#/src/cart'])
```

Comment:
similar comment to above

Reply:
https://github.com/daos-stack/daos/pull/6115/commits/539a2817b782f8c63f13ec86d9b1c360ebd469e5

Reply:
~~539a281~~  77c5ef4018424f14f53546df1d488084cada1bd0